engine/step/hook.go

```diff
@@ -0,0 +1,27 @@
+package step
+
+import (
+	"github.com/juju/errors"
+)
+
+// Hook represents the hook data structure
+type Hook struct {
+	Name            string            `json:"name"`
+	Description     string            `json:"description""`
```

Comment:
```suggestion
	Description     string            `json:"description"`
```

Typo inside the json tag